content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -239,7 +239,7 @@ sudo mkdir -p "$DOWNLOAD_DIR"
 Install crictl (required for kubeadm / Kubelet Container Runtime Interface (CRI))
 
 ```bash
-CRICTL_VERSION="v1.26.0"
+CRICTL_VERSION="v1.27.0"
```

Comment:
use `v{{ skew currentVersion }}.0` ?

Reply:
we can't use it because CRICTL versioning sometimes does not match k8s versioning. currentVersion is k8s / k/website.